phylogenetic/rules/export.smk

```diff
@@ -42,7 +42,7 @@ rule prepare_auspice_config:
     output:
         auspice_config="results/config/{gene}/auspice_config_{serotype}.json",
     params:
-        replace_clade_key="clade_membership",
+        replace_clade_key=lambda wildcard: r"clade_membership" if wildcard.gene in ['genome'] else r"nextclade_subtype",
```

Comment:
Because the `clade_membership` is automatically used to create the `clade` branch label in `augur export`, this change means the E gene build will not have the automatic `clade` branch label.

It's possible to create custom branch labels since https://github.com/nextstrain/augur/pull/728, but this uses `augur clades` to create the labels and the workflow is skipping `augur clades` for the E gene build 😅

Reply:
Since the `nextclade_subtype` field is being added through `augur traits`, maybe `augur traits` should be updated to support adding branch labels as well...

Reply:
Good to know! I'm currently planning to explore serotype and genotype-defining E mutations (which would create `clade_membership` for E gene builds) in a future PR. 

But if that doesn't work out, I'll explore "create custom branch labels ..." route you've referenced.